cromwell_tools/cromwell_tools.py

```diff
@@ -1,34 +1,144 @@
 """This module contains utility functions to interact with Cromwell.
 """
-import requests
-from requests.auth import HTTPBasicAuth
 import io
-import requests
 import zipfile
+import json
 from StringIO import StringIO
```

Comment:
just want to point out that this is not working with Python3. in python3, it might something like 'from io import StringIO'

Reply:
Yea, I can try/except this to fix, I think.